packages/flutter/lib/src/material/stepper.dart

```diff
@@ -236,6 +237,8 @@ class Stepper extends StatefulWidget {
   /// ```
   /// {@end-tool}
   final ControlsWidgetBuilder controlsBuilder;
+  /// This determines the elevation of the stepper widget when build horizontally.
```

Comment:
nit: 
```suggestion

  /// This determines the elevation of the stepper widget when built horizontally.
  ///
  /// By default, the value of `horizontalElevation` is 2.0.
```
You can add a new line, and also mention the default value used for `horizontalElevation`.

Reply:
Sorry that I forgot to mention the default value. Stupid mistake!